web-components/src/components/draggable/Draggable.ts

```diff
@@ -0,0 +1,197 @@
+/**
+ * Copyright (c) Cisco Systems, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { html, LitElement, property, PropertyValues, query } from "lit-element";
+import Sortable, { SortableOptions } from "sortablejs";
+import reset from "@/wc_scss/reset.scss";
+import styles from "./scss/module.scss";
+import { debounce } from "@/utils/helpers";
+import { customElementWithCheck, SlottedMixin } from "@/mixins";
+
+export namespace Draggable {
+  @customElementWithCheck("md-draggable")
+  export class ELEMENT extends SlottedMixin(LitElement) {
+    @property({ type: Number }) delay = 0;
+    @property({ type: Number }) animation = 0;
+    @property({ type: String }) handle = "";
+    @property({ type: String }) filter = "";
+    @property({ type: String }) easing = "";
+    @property({ type: String }) direction: "horizontal" | "vertical" = "vertical";
+    @property({ type: Object }) group: Sortable.GroupOptions | null = null;
+    @property({ type: String, attribute: "draggable-items" }) draggableItems = "md-draggable-item";
```

Comment:
these props require documentation. Is every prop here supposed to be publicly facing? or some. are supposed to be internal?

Reply:
All props presented here need to be public, because SortableJS depended on its option object